corehq/apps/userreports/sql/adapter.py

```diff
@@ -123,6 +123,11 @@ def delete(self, doc):
             delete = table.delete(table.c.doc_id == doc['_id'])
             connection.execute(delete)
 
+    def doc_exists(self, doc):
+        return self.session_helper.Session.query(
+            self.get_query_object().filter_by(doc_id=doc['_id']).exists()
```

Comment:
Checked the docs and that's correct but from what I can tell this still won't execute they query. You need `.scalar()` to actually make it execute.